src/MensaWidget/MWDataGettingWrapper.class.st

```diff
@@ -51,3 +67,83 @@ MWDataGettingWrapper class >> getTodaysMeals [
 	today := Date today yyyymmdd.
 	^ MWDataGettingWrapper getMealsOnDate: today
 ]
+
+{
+	#category : #accessing,
+	#'squeak_changestamp' : 'Oscar 7/1/2024 16:32'
+}
+MWDataGettingWrapper class >> initialLatitude [
+
+	^ InitialLatitude ifNil: [52.393822]
+]
+
+{
+	#category : #accessing,
+	#'squeak_changestamp' : 'Oscar 7/1/2024 16:21'
+}
+MWDataGettingWrapper class >> initialLatitude: anObject [
+
+	InitialLatitude := anObject
+]
+
+{
+	#category : #accessing,
+	#'squeak_changestamp' : 'Oscar 7/1/2024 16:32'
+}
+MWDataGettingWrapper class >> initialLongitude [
+
+	^ InitialLongitude ifNil: [13.129558]
+]
+
+{
+	#category : #accessing,
+	#'squeak_changestamp' : 'Oscar 7/1/2024 16:21'
+}
+MWDataGettingWrapper class >> initialLongitude: anObject [
+
+	InitialLongitude := anObject
+]
+
+{
+	#category : #accessing,
+	#'squeak_changestamp' : 'Oscar 7/1/2024 16:33'
+}
+MWDataGettingWrapper class >> initialRadius [
+
+	^ InitialRadius ifNil: [5]
+]
+
+{
+	#category : #accessing,
+	#'squeak_changestamp' : 'Oscar 7/1/2024 16:21'
+}
+MWDataGettingWrapper class >> initialRadius: anObject [
+
+	InitialRadius := anObject
+]
+
+{
+	#category : #accessing,
+	#'squeak_changestamp' : 'Oscar 7/1/2024 16:52'
+}
+MWDataGettingWrapper class >> switchLocation [
+	
+	| longitude latitude radius |
+	
+	latitude := FillInTheBlankMorph request: 'Please enter your Latitude' initialAnswer: self initialLatitude.
+	longitude := FillInTheBlankMorph request: 'Please enter your Longitude' initialAnswer: self initialLongitude.
+	radius := FillInTheBlankMorph request: 'Please enter your Radius' initialAnswer: self initialRadius.
+	self canteenDict: (MWParser transformJsonToDict: (MWApiWrapper getOutletDictFrom: latitude and: longitude andRadius: radius)).
+	self initialLongitude: longitude.
+	self initialLatitude: latitude.
+	self initialRadius: radius.
```

Comment:
but what i would like even more is something like this: 
self initialLongitude: FillInTheBlankMorph request.....
so that it become a one liner for every question asked and then still have cascades.
But we have to carful how we space it then but still feels better without extra vars then.